docs/google/engine.md

```diff
@@ -1,9 +1,17 @@
 # Quantum Engine
 
-The Quantum Engine, via the `cirq.google.Engine` class, executes programs and jobs using the
-Quantum Engine API.
-
-Note that the Quantum Engine API is not yet open for public access.
+Google's Quantum Computing Service provides the Quantum Engine API to execute 
+circuits on Google's quantum processor or simulator backends and 
+to access or manage the jobs, programs, reservations and calibrations. Cirq is 
+the only supported client for this API, using the `cirq.google.Engine` class. 
+
+Note: the Quantum Engine API is not yet open for public access.
```

Comment:
add timestamp to "not yet"

Reply:
Interesting idea - I'm guessing you're trying to avoid the situation where we open up the doors, and then this will be forgotten and will be weird? 

But what you're suggesting (timestamp / date) gets outdated immediately - i.e. for 9/15/2020 it will raise questions in the heads of users in November - is this still accurate? So to avoid that, we'd need to update these timestamps at least monthly?  I'm not convinced yet about timestamping. I think when we open the doors, we'll have a big revamp anyway and make sure the user journey's make sense, minimizing the risk of the failure scenario above.